### PR TITLE
collect vm console logs

### PR DESCRIPTION
**Pull request by nshidlin**
This PR:
- defines a console log file for VMs in maif.tf
- adds collection of console log files in the case of test failure

### CODE REVIEW

File: discovery-infra/tests/base_test.py [hunk 1]
```diff
@@ -190,8 +190,16 @@ def _collect_virsh_logs(self, nodes, log_dir_name):
...
         for node in nodes:
             infra_utils.run_command(f"cp -p /var/log/libvirt/qemu/{node.name}.log "
-                                    f"{qemu_libvirt_path}/{node.name}.log",
+                                    f"{qemu_libvirt_path}/{node.name}-qemu.log",
                                     shell=True)
+        
+        console_log_path = os.path.join(virsh_log_path, "console_logs")
+        os.makedirs(console_log_path, exist_ok=False)
+        for node in nodes:
+            infra_utils.run_command(f"cp -p /var/log/libvirt/qemu/{node.name}-console.log "
```
Comment (tsorya) [inline discovery-infra/tests/base_test.py:199]: why not to use python cp command>?

Comment (nshidlin) [reply discovery-infra/tests/base_test.py]: done

**Pull request merged**